### PR TITLE
add invokeDelete operation

### PR DESCRIPTION
**Pull request by vsaroopchand**
# Description

This proposal adds support for HTTP DELETE to invoke command. 

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #224 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ X] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: cmd/invokeDelete.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		}
+
+		print.SuccessStatusEvent(os.Stdout, fmt.Sprintf("HTTP Delete to method %s invoked successfully", invokeAppMethod))
+	},
+}
+
+func init() {
+	invokeDeleteCmd.Flags().StringVarP(&invokeAppID, "app-id", "a", "", "the app id to invoke")
+	invokeDeleteCmd.Flags().StringVarP(&invokeAppMethod, "method", "m", "", "the method to invoke")
+	invokeDeleteCmd.Flags().StringVarP(&invokeResourceID, "id", "", "", "the resource id")
```
Comment (LMWF) [inline cmd/invokeDelete.go]: What is resource id?

Comment (vsaroopchand) [reply cmd/invokeDelete.go]: The ID (key) of the resource to delete. I know Dapr refers to the State ID as Key; however, ID is traditionally used in APIs to designate the target resource.

Comment (yaron2) [reply cmd/invokeDelete.go]: Yeah, but we need to keep consistency and Dapr refers to these as methods..

Comment (vsaroopchand) [reply cmd/invokeDelete.go]: Method is the route, what about the ID of the state to delete?

This was my intention:
http://localhost:PORT/VERSION/invoke/APPID/method/**ID**

For example, if the user wants to delete state with key "99", are you suggesting they should set method as **order/99** and do away with ID?

Comment (LMWF) [reply cmd/invokeDelete.go]: Please see these docs below.  When using service invocation in dapr, only the app-id and the method name are passed in the url.  If there is data to pass it should be in the body.

https://github.com/dapr/docs/blob/master/reference/api/service_invocation.md

https://github.com/dapr/docs/blob/master/howto/invoke-and-discover-services/README.md

Comment (vsaroopchand) [reply cmd/invokeDelete.go]: Sorry, I disagree. Delete REST calls do not have a body. Please answer my question: how do I supply the KEY of the state to be deleted?

Comment (vsaroopchand) [reply cmd/invokeDelete.go]: I am going to close this PR and send a new one with my revisions.

**Pull request closed without merging**